module/gui/Collector.py

```diff
@@ -33,31 +37,53 @@ class CollectorModel(QAbstractItemModel):
         model for the collector view
     """
     
+    @classmethod
+    def time_msec(self):
+        return int((datetime.utcnow() - datetime(1970, 1, 1)).total_seconds() * 1000)
+    
     def __init__(self, view, connector):
         QAbstractItemModel.__init__(self)
-        self.connector = connector
+        self.cname = self.__class__.__name__
+        self.log = logging.getLogger("guilog")
         self.view = view
+        self.connector = connector
+        
         self._data = []
-        self.cols = 4
+        self.expandedState = {}
+        self.currentItem = None
+        self.selectedItems = []
+        self.cols = 8
         self.interval = 1
         self.mutex = QMutex()
+        self.dnd = DragAndDrop(self.view, self)
+        self.dnd.havePermissions = False
+        self.dnd.allowLinkMove = True
+        self.lastFullReload = float(0.0)
+        
+        self.dirty = False
+        self.firstSetDirty = None
+        self.lastSetDirty  = None
+        self.dirtyReloadDelayMin = float(4.5)   # const seconds
+        self.dirtyReloadDelayMax = float(9.5)   # const seconds
+        
+        self.showToolTips = True
         
         global translatedStatusMap # workaround because i18n is not running at import time
         translatedStatusMap = {
-            "finished":    _("finished"),
-            "offline":     _("offline"),
-            "online":      _("online"),
-            "queued":      _("queued"),
-            "skipped":    _("skipped"),
-            "waiting":     _("waiting"),
+            "finished":      _("finished"),
+            "offline":       _("offline"),
+            "online":        _("online"),
+            "queued":        _("queued"),
+            "skipped":       _("skipped"),
+            "waiting":       _("waiting"),
             "temp. offline": _("temp. offline"),
-            "starting":    _("starting"),
-            "failed":      _("failed"),
-            "aborted":     _("aborted"),
-            "decrypting":  _("decrypting"),
-            "custom":      _("custom"),
-            "downloading": _("downloading"),
-            "processing":  _("processing")
+            "starting":      _("starting"),
+            "failed":        _("failed"),
+            "aborted":       _("aborted"),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['_' may be undefined, or defined from star imports: PyQt4.QtCore, PyQt4.QtGui](https://app.codacy.com/manual/pyLoad/pyload/pullRequest?prid=567903)